configure.ac

```diff
@@ -2393,18 +2394,17 @@ AC_TYPE_INT64_T
 AC_TYPE_UINT64_T
 AC_TYPE_PID_T
 AC_TYPE_SIZE_T
+AC_CHECK_SIZEOF(size_t)
+SQUID_CHECK_PRINTF_CODES([size_t],[PRIuSIZE],[zu llu lu u I64u I32u])
```

Comment:
AFAICT, all master/v7 code using PRIuSIZE today is C++ code. Can we just switch that code to [std::printf/etc.](https://en.cppreference.com/w/cpp/io/c/fprintf) and `%zu` format code instead of trying to guess the right format code at ./configure time? AFAICT, doing so will improve Squid code and automatically address all other change requests.

Reply:
Hmm, good point. The odd codes were there for MSVC which we still hope to support again one day, but looking into that it was updated in 2013 to support %zu.

Reply:
Glad to hear that. Let's focus on this promising direction! We may discover that it is not working, but I think it is worth investigating.

Reply:
#1388 removes PRIuSIZE.